09_Clean_Code/praktikum/problem2_rewrite.js

```diff
@@ -0,0 +1,37 @@
+class Kendaraan {
+  constructor() {
+    this.kecepatanPerJam;
+  }
+}
+
+class Mobil extends Kendaraan {
+  constructor() {
+    super();
+    this.kecepatanPerJam = 0;
+  }
```

Comment:
Inheritance kecepatanPerJam punya class Kendaraan jadi useless, krn di class mobil this.kecepatanPerJam di declare dan di initial nilainya..

Reply:
jadi dihapus aja ya mas?